src/zenml/artifacts/utils.py

```diff
@@ -27,7 +27,8 @@
 from zenml.logger import get_logger
 from zenml.models import (
     ArtifactRequest,
-    ArtifactResponse,
+    ArtifactVersionRequest,
```

Comment:
There are several functions here that are called `load_artifact`, `save_artifact`, `load_artifact_from_response`, `log_artifact_metadata`. I was wondering if we should rename these to include `artifact_version` instead of `artifact`?

Reply:
See above